### ISSUE TITLE
New buzzwords AI Startup Cracks CAPTCHA Codes with Human-Like Vision

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#1465]
AI Startup Cracks CAPTCHA Codes with Human-Like Vision<br>
http://ift.tt/2zS9UxX<br><img width="640" height="466" src="http://ift.tt/2jnkRk4" class="webfeedsFeaturedVisual wp-post-image wp-stateless-item" style="display: block; margin: auto; margin-bottom: 20px;" data-image-size="large" data-stateless-media-bucket="cdn.thenewstack.io" data-stateless-media-name="media/2017/11/c7562b0d-vicarious-ai-2.jpg"><p>Artificial intelligence is getting more and more adept at things previously thought to be quite difficult for machines to master. But for the most part, machines still have a long way to go before they achieve human-like intelligence, yet not a week goes by without some relatively significant <a href="http://ift.tt/2lOR6nO" class="local-link">artificial intelligence</a> breakthrough making the news.</p>
<p>Now, it seems that yet another barrier of sorts has been broken. San Francisco startup <a href="http://ift.tt/2sC5zLG" target="_blank" class="ext-link" rel="external nofollow">Vicarious</a> has developed an AI that&rsquo;s capable of reliably solving <a href="http://ift.tt/Qs7KU0" target="_blank" class="ext-link" rel="external nofollow">CAPTCHA</a>, those automated security verification puzzles with squiggly or distorted lettering that you have to solve before being able to post a comment on a website, for instance.</p>
<h2>Passing the Turing Test</h2>
<p>CAPTCHA &mdash; which stands for &ldquo;Completely&nbsp;Automated&nbsp;Public&nbsp;<a href="http://ift.tt/1n9TBO1" target="_blank" class="ext-link" rel="external nofollow">Turing test</a>&nbsp;to tell&nbsp;Computers and&nbsp;Humans&nbsp;Apart&rdquo; &mdash; has been used to thwart online fraud or spam bots since their invention in the 1990s. It&rsquo;s a test that is based on an idea proposed by mathematician <a href="http://ift.tt/2sXLsHn" class="local-link">Alan Turing during the 1950s</a>, and is supposed to distinguish human from machine, since they are designed to be relatively easy for humans, but difficult for machines, to solve.</p>
<p>However, as&nbsp;Vicarious&rsquo; <a href="http://ift.tt/2zIHQJh" target="_blank" class="ext-link" rel="external nofollow">research</a> shows, it&rsquo;s now possible to tweak computer vision technology in a way that it can more closely approximate how human vision (and intelligence) works, which enables computers to solve CAPTCHAs much like how a human might. To give you an idea of a benchmark, if an algorithm can solve a CAPTCHA at least 1 percent of the time, then it&rsquo;s considered ineffective.</p>
<p>Of course, there are deep learning neural algorithms out there capable of decoding CAPTCHAs, but these previous instances have&nbsp;required a massive amount of training data to do so. In these cases, these models were trained on millions of images, such as actual strings of letters in specific fonts, rather than being able to &ldquo;generalize&rdquo; from a few training examples and recognizing individual letters under the distortions, as a human would be able to do.</p>
<div id="attachment_3430796" style="width: 1010px;" class="wp-caption aligncenter">
<img class="wp-image-3430796 size-full" src="http://ift.tt/2hvNhrw" width="1000" height="832"><p class="wp-caption-text">Comparing results of parses between Recursive Cortical Network (RCN), a Convolutional Neural Network (CNN) and two Amazon Mechanical Turk workers.</p>
</div>
<h2>Recursive Cortical Network</h2>
<p>In contrast, Vicarious&rsquo; AI was able to solve CAPTCHAs by building upon a small set of training examples. To achieve this, the company developed a type of artificial neural network called a Recursive Cortical Network (RCN), capable of doing this leap in &ldquo;generalized&rdquo; learning.</p>
<p>&ldquo;Recent AI systems like IBM&rsquo;s Watson and deep neural networks rely on brute force: connecting massive computing power to massive datasets,&rdquo; said&nbsp;Vicarious co-founder D. Scott Phoenix in a press release. &ldquo;This is the first time this distinctively human act of perception has been achieved, and it uses relatively minuscule amounts of data and computing power. The Vicarious algorithms achieve a level of effectiveness and efficiency much closer to actual human brains.&rdquo;</p>
<p>According to the company, the RCN was able to solve&nbsp;<a href="http://ift.tt/1IuMebF" target="_blank" class="ext-link" rel="external nofollow">reCAPTCHAs</a> (CAPTCHA tests consisting of scans of words from old books and newspaper, doubling as a way to digitize books) at an accuracy rate of 66.6 percent. That&rsquo;s in comparison with&nbsp;BotDetect at 64.4 percent, Yahoo at 57.4 percent and PayPal at 57.1 percent. Compared to other deep learning models for text recognition, the RCN was able to achieve equal or higher accuracy, while using only a fraction of the training data, about 300 times less.</p>
<p>The RCN is able to achieve this streamlined efficiency this by imitating a &ldquo;brain-like vision system&rdquo; like the one found in humans.</p>
<p>&ldquo;During the training phase, [the RCN] builds internal models of the letters that it is exposed to,&rdquo; as Vicarious co-founder <a href="http://ift.tt/1O9UQch" class="ext-link" rel="external nofollow">Dileep George</a> explained on <a href="http://ift.tt/2yPmxXn" target="_blank" class="ext-link" rel="external nofollow">NPR</a>. &ldquo;So if you expose it to As and Bs and different characters, it will build its own internal model of what those characters are supposed to look like. So it would say, these are the contours of the letter, this is the interior of the letter, this is the background, etc. And then, when a new image comes in &hellip; it tries to explain that new image, trying to explain all the pixels of that new image in terms of the characters it has seen before. So it will say, this portion of the A is missing because it is behind this B.&rdquo;</p>
<p><img class="aligncenter size-full wp-image-3430798" src="http://ift.tt/2hxuQCW" width="1000" height="867"></p>
<p><img class="aligncenter size-full wp-image-3430795" src="http://ift.tt/2jlU4EX" width="1000" height="927"></p>
<p>In addition to applying this model to figure out CAPTCHAs, the team has also used the RCN to identify multiple objects in images with random backgrounds, meaning that the technique could be utilized in other ways beyond the parsing of text.</p>
<p><img class="aligncenter size-full wp-image-3430797" src="http://ift.tt/2hwPcw1" width="1000" height="497"></p>
<p>This is a big step for an AI that can learn to make broader, human-like generalizations in what it sees, potentially enabling machines to better understand and therefore better interact and manipulate the world around them. But it also means that text-oriented CAPTCHAs will soon become ineffective, and better security verification measures will need to be developed, as machines become more human-like in their intelligence.</p>
<p class="attribution">Images: Vicarious</p>
<p>The post <a rel="nofollow" href="http://ift.tt/2zS9UxX">AI Startup Cracks CAPTCHA Codes with Human-Like Vision</a> appeared first on <a rel="nofollow" href="https://thenewstack.io">The New Stack</a>.</p>
<br><br>
via Blog &ndash; The New Stack https://thenewstack.io<br>
November 13, 2017 at 09:00AM
